### PR TITLE
Added additional suggestion around lineinfile

### PR DESCRIPTION
**Pull request by bontreger**
Added a recommendation to use specific modules in place of line in file. Many client playbooks use lineinfile for ssh_config and network configs

### CODE REVIEW

File: coding_style/README.adoc [hunk 1]
```diff
@@ -226,6 +226,7 @@ Try editing files directly using other built-in modules (e.g. `ini_file`, `block
 If you are modifying more than a tiny number of lines or in a manner more than trivially complex, try leveraging the `template` module, instead.
 This will allow the entire structure of the file to be seen by later users and maintainers.
 The use of `lineinfile` should include a comment with justification.
+Alternatively, most configuration files have their own modules, such as community.general.ssh_config or community.general.nmcli.  Using these make code cleaner to read and ensure idempotence.
```
Comment (ericzolf) [inline coding_style/README.adoc:229]:
```suggestion
Alternatively, most configuration files have their own modules, such as https://docs.ansible.com/ansible/latest/collections/community/general/ssh_config_module.html[community.general.ssh_config] or https://docs.ansible.com/ansible/latest/collections/community/general/nmcli_module.html[community.general.nmcli].
Using these make code cleaner to read and ensure idempotence.
```
One line per sentence.

**Comment by ericzolf** [comment]
Looks good to me, just nitpicking at format and module links.

File: coding_style/README.adoc (hunk 1 above)
Comment (richm) [reply coding_style/README.adoc:229]: Yeah, but it bothers me that we are making recommendations that Red Hat Ansible customers won't be able to use . . .

Comment (ericzolf) [reply coding_style/README.adoc]: Ah, good point, sorry I approved before seeing your remark. Could we have other examples @bontreger ?

**Comment by mophahr** [comment]
Approved in the CoP meeting

**Pull request merged**